routes/api.php

```diff
@@ -18,6 +18,4 @@
     return $request->user();
 });
 
-Route::get('/products', function () {
-    // return your computed table here
-});
+Route::get('/products', "App\Http\Controllers\ProductsController@jsonlist");
```

Comment:
Usando la sintassi più recente
```suggestion
Route::get('/products', [App\Http\Controllers\ProductsController::class, "jsonlist"]);
```